csharp/roslyn-sdk/Tutorials/MakeConst/MakeConst/MakeConstAnalyzer.cs

```diff
@@ -28,10 +28,11 @@ public class MakeConstAnalyzer : DiagnosticAnalyzer
 
         public override void Initialize(AnalysisContext context)
         {
-            // See https://github.com/dotnet/roslyn/blob/master/docs/analyzers/Analyzer%20Actions%20Semantics.md for more information
-            // <SnippetRegisterNodeAction>
+             // See https://github.com/dotnet/roslyn/blob/master/docs/analyzers/Analyzer%20Actions%20Semantics.md for more information
+            context.ConfigureGeneratedCodeAnalysis(GeneratedCodeAnalysisFlags.Analyze | GeneratedCodeAnalysisFlags.None);
+            context.EnableConcurrentExecution();
             context.RegisterSyntaxNodeAction(AnalyzeNode, SyntaxKind.LocalDeclarationStatement);
-            // </SnippetRegisterNodeAction>
```

Comment:
This closing snippet tag cannot be deleted. There has to be one closing snippet tag to correspond to every opening snippet tag. They're how the build system knows to import snippets within larger source code files.

Reply:
Same issue as above. It should be fixed.